pages/stats.js

```diff
@@ -61,7 +61,7 @@ export default function Stats({ error, stats }) {
             {stats.countByUser.map((user, index) => {
               return (
                 <tr className="border-t">
-                  <td className="py-2 md:pl-10">{user.name}</td>
+                  <td className="py-2 md:pl-10"><a href={user.html_url} target="_blank">{user.name}</a></td>
```

Comment:
Actually... what is html_url from? That's in the Github endpoint but not in the stats object I think... I'll test it locally.

Reply:
iirc i don't actually have the object to view (its not public), it seems to be on the private database of users. it was really just a shot in the dark, seeing that there were other similarly named fields in the github endpoint

Reply:
I see. Well to be honest I never got to test it so I'll check it out next october :P.